templates/repo/editor/patch.tmpl

```diff
@@ -0,0 +1,59 @@
+{{template "base/head" .}}
+<div class="page-content repository file editor edit">
+	{{template "repo/header" .}}
+	<div class="ui container">
+		{{template "base/alert" .}}
+		<form class="ui edit form" method="post" action="{{.RepoLink}}/_diffpatch/{{.BranchName  | PathEscapeSegments}}">
+			{{.CsrfTokenHtml}}
+			<input type="hidden" name="last_commit" value="{{.last_commit}}">
+			<input type="hidden" name="page_has_posted" value="{{.PageHasPosted}}">
```

Comment:
Here and other files, I didn't find the usage of `page_has_posted` and didn't find the definition of `PageHasPosted`

Reply:
https://github.com/go-gitea/gitea/blob/f8b21ac04a02a784fe14941e5c89940448694b8c/web_src/js/features/repo-editor.js#L148